mm2src/coins/tendermint/tendermint_token.rs

```diff
@@ -494,10 +494,10 @@ impl MmCoin for TendermintToken {
                 .any_to_transaction_data(maybe_priv_key, msg_payload, &account_info, fee, timeout_height, &memo)
                 .map_to_mm(|e| WithdrawError::InternalError(e.to_string()))?;
 
-            let internal_id = {
-                let hex_vec = tx.tx_hex().cloned().unwrap_or_default().to_vec();
-                sha256(&hex_vec).to_vec().into()
-            };
+            let internal_id = super::tendermint_tx_internal_id(
+                tx.tx_hex().cloned().unwrap_or_default().to_vec(),
+                Some(token.token_id()),
+            );
```

Comment:
wouldn't this create any backward compatibility issues.

Reply:
All we do is printing the value on CLI, so I don't think this will cause BC.

Reply:
aha i see. i confused that with the internal_id in `tendermint_tx_history_v2.rs` (which persists the internal_id in a database).

i see both internal_ids aren't related tho? no?

Q: is that internal_id returned from the RPC of any use to the user? it doesn't carry any particularly interesting info ha?